packages/wonder-blocks-core/util/types.js

```diff
@@ -100,6 +100,15 @@ export type AriaProps = {|
     "aria-busy"?: "true" | "false",
     "aria-checked"?: "false" | "mixed" | "true" | "undefined",
     "aria-controls"?: IdRefList,
+    // https://www.w3.org/TR/wai-aria-1.1/#aria-current
+    "aria-current"?:
+        | "false"
```

Comment:
TIL, thanks for linking to the spec.

Reply:
Did I completely miss `"aria-current"`?  That's embarrassing. 🙈

Reply:
You don't have to 😃. I checked the original link used for reference (https://www.w3.org/WAI/PF/aria-1.1/roles#role_definitions) and they don't include `aria-current` there.

I'm going to create a ticket to check if there are more attributes that would be good to add!

Reply:
https://khanacademy.atlassian.net/browse/WB-611